baip-0007.md

```diff
@@ -0,0 +1,58 @@
+BAIP: 0007
+Title: Proposal on bitASSETS reward and BTS destruction
+Authors: cn-vote   zhouxiaobao-2010
+Status: Accepted
+Type: Consensus
+Created: 2020-06-01
+
+# Abstract
+The purpose of this proposal is to establish a scientific financial model of BTS, make reasonable use of the commission income of the committee, open up resources and reduce expenditure, and promote the stable growth of BTS system value.
+
+# Specification
+
+### bitASSETS reward
+
+Intelligent assets are the original assets of the system, and the corresponding holders should enjoy the dividend of system growth.
+Reward amount: 20000 BTS per day
+Participating assets: bitCNY, bitUSD, bitEUR, bitRUBLE
+
+1.Reward conditions:
+1)The holding amount should exceed X. For example: those over 1000 yuan can participate in reward.
+2)The holding amount needs to last for a period of time . For example, 7 days. Rewards can only be distributed if the holding amount meets 1000bitcny or above for 7 consecutive days.
```

Comment:
Are these rules supposed to be dynamic? Will committee adjust them?

Reply:
The basic principle is fixed, but the parameters may be adjusted according to the actual situation. There are rules on how to adjust the parameters after the proposal. For example, in the future, if there is not enough commission income in the commission account, the amount of reward will be reduced proportionally.